CanDIG/candig-opa
# v2.1.0: OPA Service tokens and health check

## v4.0.0: Description
* Add a path for Opa to verify service tokens
* Add a health check
* Add HTSGet paths

- [x] Tagged as a release in this repo
- [ ] Passes integration tests on a development instance
- [ ] Images pushed to Dockerhub
